Sources/SwiftParser/Directives.swift

```diff
@@ -106,15 +106,21 @@ extension Parser {
       while let poundIfHandle = firstIteration ? self.canRecoverTo(.poundIfKeyword) : self.canRecoverTo(anyIn: IfConfigContinuationClauseStartKeyword.self)?.handle,
         loopProgress.evaluate(self.currentToken)
       {
-        let (unexpectedBeforePoundIf, poundIf) = self.eat(poundIfHandle)
+        var (unexpectedBeforePoundIf, poundIf) = self.eat(poundIfHandle)
         firstIteration = false
         // Parse the condition.
         let condition: RawExprSyntax?
         switch poundIf.tokenKind {
         case .poundIfKeyword, .poundElseifKeyword:
           condition = RawExprSyntax(self.parseSequenceExpression(.basic, forDirective: true))
         case .poundElseKeyword:
-          condition = nil
+          if let ifToken = self.consume(if: .init(.if, allowAtStartOfLine: false)) {
```

Comment:
@ahoppen I had to change this line to use the new `consume` API (allowsAtStartOfLine got removed as a parameter). Is this method usage correct?